app/models/basket.rb

```diff
@@ -0,0 +1,5 @@
+class Basket < ApplicationRecord
+  belongs_to :user
+  has_many :subscriptions
```

Comment:
Does this needs dependent destroy to delete subscriptions if basket is deleted

Reply:
Agreed, will add dependency